File: TiendaAPI/Entities/Carro.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Threading.Tasks;

namespace TiendaAPI.Entities
{
    public class Carro
    {
  
        [Key]
        public int idCarro { get; set; }
        
        public int idUsuario { get; set; }
        public bool activo { get; set; }

        [InverseProperty("Carro")]
        public virtual ICollection<Producto_Carro> Prod_Carr { get; set; }

    }
}

File: TiendaApp/src/app/models/carro.ts
import { ProductoCarro } from './producto-carro';
export class Carro {

    idCarro:number;
    idUsuario:number;
    activo:boolean
    prod_Carr:ProductoCarro[]
    
    constructor(idCarro:number,idUsuario:number,activo:boolean,prod_Carr:ProductoCarro[]){
        this.idCarro = idCarro;
        this.idUsuario = idUsuario;
        this.activo = activo;
        this.prod_Carr = prod_Carr;
    }


}
